### DISCUSSION TITLE
Simple clap plugin bridge to fluidsynth.

### DISCUSSION THREAD

**Discussion by dbadb** [free-audio/clap#329] [Show and tell]
Why not have access to soundfonts in your clap host?

https://github.com/cannerycoders/fluidsynth.clap

Windows builds available today, MacOS and Linux in a few more days.

(plugin is MIT licensed, open source, etc)

**Comment by dbadb** [comment]
As promised, the MacOS and Linux builds are now available via the 0.2.0 release.

**Comment by JBetz** [comment]
Thinking about how to communicate Soundfont preset names to hosts.

What about using `params->value_to_text`?

So, for all the program parameters, the plugin would use `fluid_sfont_get_preset` and `fluid_preset_get_name` with the current bank and program number.

**Reply by dbadb** [reply]
There is an interface: `clap/factory/draft/preset-discovery.h` that may address this requirement?
That said, soundfonts might be stored anywhere on a filesystem beyond some standard locations.
